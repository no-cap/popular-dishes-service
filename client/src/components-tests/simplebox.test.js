import React from 'react';
import { shallow } from 'enzyme';
import SimpleBox from '../components/simplebox.jsx';
describe('Components should render correctly', () => {
  test('SimpleBox should render correctly in "debug" mode', () => {
    const component = shallow(<SimpleBox myValue={5} />);

    expect(component).toMatchSnapshot();
  });
});