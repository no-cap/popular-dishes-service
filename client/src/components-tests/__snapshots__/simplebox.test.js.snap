// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Components should render correctly SimpleBox should render correctly in "debug" mode 1`] = `ShallowWrapper {}`;
